src/project-tests/pomodoro-clock-tests.js

```diff
@@ -78,476 +84,718 @@ export default function createPomodoroClockTests() {
+        clickButtonsById(Array(30).fill(_seshMin));
+        assert.strictEqual(
+          document.getElementById('session-length').innerHTML,
+          '1',
+          'Value in element with id of "session-length" is less than 1.'
+        );
       });
 
-      it('17. I should not be able to set a session or break length to > 60.', function() {
-        clickButtonsById(Array(60).fill(_break_plus));
-        FCC_Global.assert.strictEqual(document.getElementById("break-length").innerHTML, "60", 'Value in element with ' + 'id of "break-length" is greater than 60.');
+      reqNum++;
+      it(`${reqNum}. I should not be able to set a session or break length to 
+      > 60.`, function() {
+        clickButtonsById(Array(60).fill(_breakPlus));
+        assert.strictEqual(
+          document.getElementById('break-length').innerHTML,
+          '60',
+          'Value in element with id of "break-length" is greater than 60.'
+        );
         resetTimer();
-        clickButtonsById(Array(40).fill(_sesh_plus));
-        FCC_Global.assert.strictEqual(document.getElementById("session-length").innerHTML, "60", 'Value in element with ' + 'id of "session-length" is greater than 60.');
+        clickButtonsById(Array(40).fill(_seshPlus));
+        assert.strictEqual(
+          document.getElementById('session-length').innerHTML,
+          '60',
+          'Value in element with id of "session-length" is greater than 60.'
+        );
       });
 
-      it('18. When I first click the element with id="start_stop", the timer should begin running from the value currently displayed in id="session-length", even if the value has been incremented or decremented from the original value of 25.', function() {
-        clickButtonsById([_start_stop]);
-        FCC_Global.assert.strictEqual(getMinutes(document.getElementById("time-left").innerHTML), document.getElementById("session-length").innerHTML);
+      reqNum++;
+      it(`${reqNum}. When I first click the element with id="start_stop", the 
+      timer should begin running from the value currently displayed in 
+      id="session-length", even if the value has been incremented or 
+      decremented from the original value of 25.`, function() {
+        clickButtonsById([_startStop]);
+        assert.strictEqual(
+          getMinutes(
+            document.getElementById('time-left').innerHTML
+          ),
+          document.getElementById('session-length').innerHTML
+        );
       });
 
-      it('19. If the timer is running, the element with the id of "time-left" should display the remaining time in mm:ss format (decrementing by a value of 1 and updating the display every 1000ms).', function() {
+      reqNum++;
+      it(`${reqNum}. If the timer is running, the element with the id of 
+      "time-left" should display the remaining time in mm:ss format 
+      (decrementing by a value of 1 and updating the display every 1000ms).`,
+      function() {
```

Comment:
This is an example of `function() {` on its own line. This is the formatting we should be using.